roles/test_metrics_retention/tasks/metrics_retention_test.yml

```diff
@@ -0,0 +1,17 @@
+---
+# Assuming we've already logged in...
+# Following procedure on https://infrawatch.github.io/documentation/#creating-an-alert-rule-in-prometheus_assembly-advanced-features
+# Assuming we're in teh right project already...
+
```

Comment:
Personally, I'd add a pre-check to make sure the metrics retention is not already set to 7d, to make sure this actually sets something and/or the expected pre-conditional are met

Reply:
We are adding a whole section that includes prometheus in case it doesn't exist at all.
I will change 7d to 17d. I am pretty sure we won't hit deployment with retention 17 days.